dev_guide/image_pull_secrets.adoc

```diff
@@ -0,0 +1,94 @@
+= Image Pull Secrets
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+link:../architecture/infrastructure_components/image_registry.html[Docker
+registries] can be secured to prevent unauthorized parties from accessing
+certain images. If you are using OpenShift's integrated docker registry and are
+pulling images for pods located in the same project, then your service account
```

Comment:
...pulling from image streams located in the same project, then your pod's service account...